src/anim/anim-target.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @private
+ * @callback pc.AnimSetter
+ * @description Callback function that the {@link pc.AnimEvaluator} uses to set final animation values.
+ * These callbacks are stored in {@link pc.AnimTarget} instances which are constructed by an
+ * {@link pc.AnimBinder}.
+ * @param {number[]} value - updated animation value.
+ */
```

Comment:
is there some better place for this one?

Reply:
It could arguably go in `src\callbacks.js`. But we can consider that in a follow up PR.